internal/provider/schema_gitlab_project_issue.go

```diff
@@ -0,0 +1,364 @@
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Optional:    true,
+		},
+		"moved_to_id": {
+			Description: "The id of the issue that was moved to.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+		},
+		"upvotes": {
+			Description: "The number of upvotes the issue has received.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+		},
+		"downvotes": {
+			Description: "The number of downvotes the issue has received.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+		},
+		"web_url": {
+			Description: "The web URL of the issue.",
+			Type:        schema.TypeString,
+			Computed:    true,
+		},
+		// NOTE(TF): only the `full` reference is used here,
+		//           clarify first what to do with nested types.
+		"references": {
+			Description: "The references of the issue.",
+			Type:        schema.TypeString,
+			Computed:    true,
+			Optional:    true,
+		},
+		// NOTE(TF): these are from the `time_stats` field.
+		//           Clarify what to do with nested types.
+		"time_estimate": {
+			Description: "The time estimate of the issue.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Optional:    true,
+		},
+		"total_time_spent": {
+			Description: "The total time spent of the issue.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Optional:    true,
+		},
+		"human_time_estimate": {
+			Description: "The human-readable time estimate of the issue.",
+			Type:        schema.TypeString,
+			Computed:    true,
+			Optional:    true,
+		},
+		"human_total_time_spent": {
+			Description: "The human-readable total time spent of the issue.",
+			Type:        schema.TypeString,
+			Computed:    true,
+			Optional:    true,
+		},
+		// NOTE(TF): end `time_stats`
+		// NOTE: not part of `CREATE`, but part of `UPDATE`
+		"discussion_locked": {
+			Description: "Whether the issue is locked for discussions or not.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+		},
+		"subscribed": {
+			Description: "Whether the authenticated user is subscribed to the issue or not.",
+			Type:        schema.TypeBool,
+			Computed:    true,
+		},
+		"user_notes_count": {
+			Description: "The number of user notes on the issue.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+		},
+		"links": {
+			Description: "The links of the issue.",
+			Type:        schema.TypeMap,
+			Elem:        &schema.Schema{Type: schema.TypeString},
+			Computed:    true,
+			Optional:    true,
+		},
+		"issue_link_id": {
+			Description: "The id of the issue link.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+		},
+		"merge_request_count": {
+			Description: "The number of merge requests associated with the issue.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+		},
+		"epic_issue_id": {
+			Description: "The id of the epic issue.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Optional:    true,
+		},
+		"epic_id": {
```

Comment:
Isn't there a comment on L83-89 saying this isn't supported?

Reply:
good catch ;)